### PR TITLE
TECH-2820-NervesHub-remote-console-setup

### PR DESCRIPTION
**Pull request by staffdill**

### CODE REVIEW

File: modules/api/main.tf [hunk 1]
```diff
@@ -418,6 +418,56 @@ DEFINITION
...
+  memory                   = var.memory
+
+  tags = var.tags
+
+  container_definitions = <<DEFINITION
+   [
+     ${module.logging_configs.fire_lens_container},
+     ${module.logging_configs.datadog_container},
+     {
+       "portMappings": [
```
Comment (rraub) [inline modules/api/main.tf]: Do we need ports open for a remote console? I don't think so... but happy to be overruled.

**Review by rraub — CHANGES_REQUESTED** [review]
I think theres a few more changes we'll need to make here, off the top of my head permissions to enable this on the cluster or service need to be enabled 
https://aws.amazon.com/blogs/containers/new-using-amazon-ecs-exec-access-your-containers-fargate-ec2/
Also we should make a `remote-console.sh` script https://github.com/smartrent/control.smartrent.com/blob/main/infrastructure/scripts/remote-console.sh or do you think that will go in another repo? 

Also could you add the ticket number on the front of this PR so I can find it in Jira 🙇

File: modules/api/main.tf (hunk 1 above)
Comment (staffdill) [reply modules/api/main.tf]: > I think theres a few more changes we'll need to make here, off the top of my head permissions to enable this on the cluster or service need to be enabled https://aws.amazon.com/blogs/containers/new-using-amazon-ecs-exec-access-your-containers-fargate-ec2/ Also we should make a `remote-console.sh` script https://github.com/smartrent/control.smartrent.com/blob/main/infrastructure/scripts/remote-console.sh or do you think that will go in another repo?
> 
> Also could you add the ticket number on the front of this PR so I can find it in Jira 🙇

updated the name didnt see that thank you, and Ill take a look at the permissions and yeah as for the remote console script its going in this repo here: https://github.com/smartrent/nerves_hub_infrastructure/pull/131

**Comment by rraub** [comment]
Have we applied this in dev so we can test it out?

**Comment by rraub** [comment]
@staffdill when your ready, re-request a review from me on github show it shows up in my feed. 🙇

File: modules/api/main.tf (hunk 1 above)
Comment (staffdill) [reply modules/api/main.tf]: removed the ports and resolved all the issues with this. Permissions are updated as well and tested for all 3 services

**Review by rraub — APPROVED** [review]
I was able to run this successfully in dev! thank you!

We will need to make sure that we're updating this task definition when we are doing deployments (responsibility of another repo)

**Pull request merged**